momentjs-rails.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{lib,vendor}/**/*"] + ["news.md", "MIT-LICENSE", "README.md"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "railties", ">= 3.1"
-  s.add_development_dependency "rails", "~> 3.2.12"
-  s.add_development_dependency 'test-unit', '~> 3.0'
+  s.add_dependency "railties", ">= 4.2"
```

Comment:
@derekprior, @jeperkins4: Would be nice to get this merged, was about to fork and pull request update as well :)